packages/config/config/devices/0x0460/qnsw-001P16.json

```diff
@@ -250,6 +255,7 @@
 		{
 			"#": "201",
 			"label": "Serial Number: Part 1",
+			"description": "This parameter contains a part of devices serial number.",
```

Comment:
IMO these descriptions are redundant with the label.

Reply:
Correct, will remove it in the next update, I just wanted to get trough with the first one, so that I can prepare config files for the rest devices.

Reply:
Works for me. Let me know if you're done with this one. The test failure is due to a formatting error I can fix.